### PR TITLE
Update Pydantic V1 to 1.10.13

### PR DESCRIPTION
**Pull request by hramezani**

### CODE REVIEW

File: pydantic/v1/datetime_parse.py [hunk 1]
```diff
@@ -147,7 +147,7 @@ def parse_time(value: Union[time, StrBytesIntFloat]) -> time:
     number = get_numeric(value, 'time')
     if number is not None:
         if number >= 86400:
-            # doesn't make sense since the time loop back around to 0
+            # doesn't make sense since the time time loop back around to 0
```
Comment (samuelcolvin) [inline pydantic/v1/datetime_parse.py:150]: typo

Comment (lig) [reply pydantic/v1/datetime_parse.py:150]: We need to fix it in v1 first:)

**Review by lig — APPROVED** [review]
👍🏻

File: pydantic/v1/datetime_parse.py (hunk 1 above)
Comment (hramezani) [reply pydantic/v1/datetime_parse.py:150]: It got merged. we need to fix it in `v1`

**Pull request merged**